Testing/SystemTests/tests/framework/ISIS/SANS/WORKFLOWS/SANSSingleReductionTest.py

```diff
@@ -78,11 +78,11 @@ def _compare_to_reference(self, workspace, reference_file_name, check_spectra_ma
         load_name = "LoadNexusProcessed"
         load_options = {"Filename": reference_file_name, "OutputWorkspace": EMPTY_NAME}
         load_alg = create_unmanaged_algorithm(load_name, **load_options)
-        load_alg.setProperty("OutputWorkspace", reference_file_name.split(".")[0])
```

Comment:
Is there any reason this needs to be overridden form the base class? Won't the call to `Load` select the correct loader for processed nxs?